MANIFEST

```diff
@@ -2309,6 +2309,21 @@ cpan/Term-ANSIColor/t/module/true-color.t
 cpan/Term-ANSIColor/t/taint/basic.t
 cpan/Term-Cap/Cap.pm			Perl module supporting termcap usage
 cpan/Term-Cap/test.pl			See if Term::Cap works
+cpan/Term-Table/lib/Term/Table.pm		Term-Table
+cpan/Term-Table/lib/Term/Table/Cell.pm		Term-Table
+cpan/Term-Table/lib/Term/Table/CellStack.pm	Term-Table
+cpan/Term-Table/lib/Term/Table/HashBase.pm	Term-Table
```

Comment:
Shouldn't this be done for the Win32 makefiles too?

Similarly for the second commit.

Reply:
Urr, wrong chunk